WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -238,9 +238,9 @@ private extension StoreStatsAndTopPerformersViewController {
                     DDLogError("⛔️ Error synchronizing order stats: \(error)")
                     periodSyncError = error
                 }
-                group.leave()
                 periodGroup.leave()
                 periodStoreStatsGroup.leave()
+                group.leave()
```

Comment:
nit: maybe it's worth making a comment so that we don't accidentally reorder it?

Reply:
good thought!